Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -354,6 +525,42 @@ bool isFunctionJson(string fullName)
             }
         }
 
+        private async Task LocalAppFetch(IRepository repository, DeploymentInfoBase deploymentInfo, string targetBranch, ILogger logger, ITracer tracer)
```

Comment:
>LocalAppFetch [](start = 27, length = 13)

Can you confirm that d:\home\site\deployments\active file is updated by onedeploy for all scenarios? legacy scenarios like wardeploy/zipdeploy and also for new scenarios like app.war, static, etc

Reply:
Yeah, I can confirm that the active file if updated by OneDeploy for all scenarios